stock_release_channel_shipment_advice_process_end_time/models/res_config_settings.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+
+    _inherit = "res.config.settings"
+
+    release_channel_process_end_time_delay = fields.Integer(
```

Comment:
@sbejaoui Into the ui we've a text to describe the meaning of this field but the name doesn't reflect its usage